patches/components-crash-content-app-crashpad_linux.cc.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/components/crash/content/app/crashpad_linux.cc b/components/crash/content/app/crashpad_linux.cc
+index 05767fe2a26e16a0152ba419ed38a944f919cbde..47c474ed3221583973d16456443902c83e133ae4 100644
+--- a/components/crash/content/app/crashpad_linux.cc
++++ b/components/crash/content/app/crashpad_linux.cc
+@@ -143,6 +143,8 @@ bool BuildHandlerArgs(base::FilePath* handler_path,
+ 
+ #if defined(GOOGLE_CHROME_BUILD) && defined(OFFICIAL_BUILD)
+   *url = "https://clients2.google.com/cr/report";
++#elif defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
```

Comment:
Can we use `OnBeforeURLRequest_StaticRedirectWork` for this instead of patching?

Reply:
I tried to use `OnBeforeURLRequest_StaticRedirectWork` at first, but from what I could see it's not consulted at all by crashpad (for example, the Windows version of the crashpad client just makes a direct WinHTTP connection to the specified server).

Reply:
Right, it's coming from the client, but some of these have other ways to override. I know not all of them do, but at least some do

Reply:
Ya, we went over ways to get rid of all the .cc and .h patching today.

Reply:
CHROME_CRASHPAD_SERVER_URL for windows, not sure about the others

Reply:
We can add it to the env in the BraveMainDelegate initializer, but I'm also fine with chromium_src patching if that can be done cleanly. The env may only work for windows, but windows is also the biggest pain because of the shared module.

Reply:
OK, gotcha. I'll see where I get with the chromium_src patching and if it doesn't turn out to be feasible for some reason, I'll go with the approach you're suggesting. Thanks for the tip!

Reply:
The chromium_src patching for this one has gotten fairly complicated and I still don't have it working. The main hindrance is that `GetCrashpadClient` isn't an easily-replaceable free function as in some of our other replacement scenarios; it's defined as part of an interface in a .h file, which has led to clashes due to differing return values on the existing function vs. its replacement.

Since that approach got a little more complicated than seemed reasonable, I started to investigate taking advantage of `CHROME_CRASHPAD_SERVER_URL` on Windows. It doesn't seem that this will work, however, as Chrome uses an ELF (Early Loading Framework) DLL to perform crash reporting on Windows. This means that the crash reporter loads before anything else (including WinMain), so it's impossible to inject anything into its environment.

I think it may make sense to patch these for now and revisit later.